src/__tests__/incidents/Incidents.test.tsx

```diff
@@ -45,43 +46,39 @@ describe('Incidents', () => {
   describe('routing', () => {
     describe('/incidents/new', () => {
       it('The new incident screen when /incidents/new is accessed', () => {
-        const { container } = setup([Permissions.ReportIncident], '/incidents/new')
-
-        expect(container).toHaveTextContent('incidents.reports')
+        setup([Permissions.ReportIncident], '/incidents/new')
+        expect(screen.getAllByText(/incidents.reports/i)[0]).toBeInTheDocument()
```

Comment:
there is only 1 `expectedIncident` => can we make getByText assertion by some field that is only visible once?